src/Heatmap/HeatMapInterpolationPoints.swift

```diff
@@ -0,0 +1,399 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import Foundation
+import GooglePlaces
+import GoogleMaps
+import GoogleMapsUtils
+
+/// This class will create artificial points in surrounding locations with appropriate intensities interpolated by neighboring intensity values.
+class HeatMapInterpolationPoints {
```

Comment:
The class needs to be added to the xcode project file for it to be included in the library, i don't think that file was modified?

Reply:
Do you mean GoogleMapsUtils.xcodeproj? I only see a couple files in there and they're all in raw formatting; I only copied my heat map class and copied it into the samples library and the unit tests into the tests library.